site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -40,12 +40,22 @@ of type [Ruleset][4], as shown below.
 ## Rule Resources
 
 An extension can specify up to [MAX_NUMBER_OF_STATIC_RULESETS][5] [rulesets][6] as part of the
-`"rule_resources"` manifest key. The number of rules across **enabled** static rulesets must be less
-than the [MAX_NUMBER_OF_RULES][7] constant.
+`"rule_resources"` manifest key. An extension is allowed to enable up to at least
+[GUARANTEED_MINIMUM_STATIC_RULES][7] static rules. Additional static rule sets may or may not be
+enabled depending on the available [global static rule limit][8].
+
+## Global Static Rule Limit
+
+In addition to the [GUARANTEED_MINIMUM_STATIC_RULES][9] static rules guaranteed for each extension,
+extensions can enable additional static rulesets depending on the available global static rule limit. This
+global limit is shared between all extensions and can be used by extensions on a first come first
+serve basis. Extensions shouldn't depend on the global limit having a specific value and should
+instead rely the [getAvailableStaticRuleCount][10] API method to find the additional rule limit available to
+them.
 
 ## Rules
 
-A single declarative [Rule][8] consists of four fields: `id`, `priority`, `condition` and `action`.
+A single declarative [Rule][11] consists of four fields: `id`, `priority`, `condition` and `action`.
```

Comment:
`condition`, and
https://developers.google.com/style/commas#serial-commas

Reply:
done